app/components/Session.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import NewSessionForm from './Session/NewSessionForm';
+import SavedSessions from './Session/SavedSessions';
+import { Tabs, Form, Input, Button, Spin } from 'antd';
+import { ServerTypes } from '../actions/Session';
+import './Session.css';
+
+const { TabPane } = Tabs;
```

Comment:
for things other than import/exports, we don't use spaces around braces. so this should be:

```js
const {TabPane} = Tabs;
```

(we do things this way to distinguish semantically between imports/exports and simple destructured assignment)